bigbluebutton-html5/imports/ui/components/user-list/service.js

```diff
@@ -210,8 +211,23 @@ const getCurrentUser = () => {
   return (currentUser) ? mapUser(currentUser) : null;
 };
 
+const isMeetingLocked = () => {
+  const meeting = Meetings.findOne({});
+  const lockSettings = meeting.lockSettingsProp;
+
+  if ( lockSettings.disableCam 
```

Comment:
At the start of the meeting, if there has been no lock settings yet on the meeting, the userlist won't open, because these properties are undefined.